### PR TITLE
docs: install obf-new on off2

### PR DESCRIPTION
**Pull request by stephanegigandet**
Work in progress: installing a new instance of OBF on off2, with the new code.

The instance can be used for testing first, and then become the production instance.

On off2, obf-new is container 116.

### CODE REVIEW

File: docs/reports/2024-04-26-off2-obf-new-install.md [hunk 1]
```diff
@@ -621,67 +655,52 @@ JPEG: Test code failed: Can't link/include 'jpeglib.h', 'jpeg'
...
 ```
 
 
+
+
+
 ### Apache
 
-On obf and opf we start by removing default config and disabling mpm_event in favor of mpm_prefork, and change logs permissions
+>e start by removing default config and disabling mpm_event in favor of mpm_prefork, and change logs permissions
```
Comment (teolemon) [inline docs/reports/2024-04-26-off2-obf-new-install.md:703]:
```suggestion
We start by removing default config and disabling mpm_event in favor of mpm_prefork, and change logs permissions
```

File: docs/reports/2024-04-26-off2-obf-new-install.md [hunk 2]
```diff
@@ -0,0 +1,1104 @@
+# 2023-06-07 New install of OBF on OFF2 with new generic code
+
+## Introduction
```
Comment (alexgarel) [inline docs/reports/2024-04-26-off2-obf-new-install.md]:
```suggestion
## Introduction
```

File: docs/reports/2024-04-26-off2-obf-new-install.md [hunk 3]
```diff
@@ -0,0 +1,1104 @@
...
+adduser --uid 1000 off
+```
+
+I also create a user *stephane*:
+
+```bash
+adduser stephane
+```
+
+### Install sudo, add your user to sudo group
```
Comment (alexgarel) [inline docs/reports/2024-04-26-off2-obf-new-install.md]:
```suggestion
### Install sudo, add your user to sudo group
```

**Review by alexgarel — APPROVED** [review]
Thanks for the thorough documentation !

**Pull request merged**